js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+            'credited': 'ok',
+            'confirmed': 'ok',
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    async fetchDeposit (id, code = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposit() requires a txid argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'txid': id,
+        };
+        const response = await this.privateGetDeposit (this.extend (request, params));
+        // {
+        //   "id": 4831985,
+        //   "currency": "ltc",
+        //   "amount": "0.14108256",
+        //   "fee": "0.0",
+        //   "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //   "uid": null,
+        //   "created_at": "2022-02-01T18:10:32+03:00",
+        //   "confirmations": "6",
+        //   "done_at": null,
+        //   "state": "accepted"
+        // }
+        // const data = this.safeValue (response, 'data', {});
+        // const deposit = this.safeValue (data, 'deposit', {});
+        return this.parseTransaction (response);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+        }
+        const response = await this.privateGetDeposits (params);
+        // const data = this.safeValue (response, 'data', {});
+        // const deposits = this.safeValue (response);
+        // [
+        //     {
+        //         "id": 4831985,
+        //         "currency": "ltc",
+        //         "amount": "0.14108256",
+        //         "fee": "0.0",
+        //         "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //         "uid": null,
+        //         "created_at": "2022-02-01T18:10:32+03:00",
+        //         "confirmations": "6",
+        //         "done_at": null,
+        //         "state": "accepted"
+        //     }
+        // ]
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    parseOrder (order, market = undefined) {
+        // [
+        //   {
+        //     "id": 287792646,
+        //     "at": 1643956089,
+        //     "side": "sell",
+        //     "ord_type": "limit",
+        //     "price": "0.141",
+        //     "avg_price": "0.0",
+        //     "state": "cancel",
+        //     "market": "ltcbtc",
+        //     "created_at": "2022-02-04T09:28:09+03:00",
+        //     "volume": "0.141",
+        //     "remaining_volume": "0.141",
+        //     "executed_volume": "0.0",
+        //     "trades_count": 0,
+        //     "strategy": null
+        //   }
+        // ]
+        const id = this.safeString (order, 'id');
+        const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
```

Comment:
You have already a `timestamp` and a `datetime` in the response, so there is no need to convert one into the other

Do this instead:
```
const timestamp = this.safeTimestamp(order, 'at');
const datetime = this.safeString(order, 'created_at');
```
![image](https://user-images.githubusercontent.com/43336371/153574248-0e510227-2216-423c-bf41-5814f6a6fd3d.png)

Reply:
In general, we prefer parsing the timestamp and generating the datetime from the integer timestamp. Also, in this case we should parse the timestamp, but generate the datetime from the timestamp:

```
const timestamp = this.safeTimestamp (order, 'at');
return this.safeOrder ({
    // ...
    'timestamp': timestamp,
    'datetime': this.iso8601 (timestamp), 
    // ...
}, market);
```

That is because we must return UTC-dates, but the exchanges can format their datetimes how they want. We don't rely on their format, instead we generate our datetime format from the timestamp with iso8601.

Reply:
Ok my bad then 😬  disregard my comment